### ISSUE TITLE
Spotify Freezing

### ISSUE THREAD

**Issue by zabinga73** [bermudalocket/MeltedCrayons#20]
**Describe the bug**
When skipping a song, there is a high chance of the app freezing while changing to the next song. The app may also freeze at random while playing in the background. Freezing does not stop the music, it just makes the app unresponsive

**To Reproduce**
1. Go to Spotify
2. Play a song
3. Either Skip the song, which may take a few tries, or allow the app to play music in the background for a while.
4. See Freeze

**Expected behavior**
1.A smooth transition to the next song in the case of the skipping freeze.
2.music to play in the background and for the app to continue to be responsive after reopening it.

**Screenshots**
![6AC5DF0B-E70A-43E3-B253-131A22B5B76E](https://user-images.githubusercontent.com/62087923/76540411-56312500-643f-11ea-8c7c-84101ecc60a9.png)

Notice how there is a black rectangle on the side, possibly the album artwork not loading fast enough? (was switching from Bad Decisions by The Strokes to St. Elmo’s Fire by John Parr, in case it matters)

**Smartphone:**
 - Device: iPhoneXR
 - OS: iOS13.3 (unc0ver v4.3.1)

**Additional context**
-Disabled and Uninstalled Groovify and Sposify, neither fixed the problem
-Tried on Spotify v8.5.48 (newest version) and on Spotify v8.5.44
-Restarting the app after a freeze happens causes the tweak to be essentially ‘disabled’, with a color relative to the album artwork being shown, similar to Colorflow. (note that Colorflow is disabled on Spotify as I have this issue). Needs to be restarted again to be able to see the tweak again.

**Comment by bermudalocket** [comment]
Very descriptive and well-written bug report. Thank you for making my job easier!

**Comment by bermudalocket** [comment]
Hopefully fixed in v1.0.1.

**Comment by zabinga73** [comment]
Very glad to say that the issue is fixed on my end! Great job on the update. I noticed that you have changed the algorithm for how the background is made. example is then:
![7B1CB75D-AC2A-4455-9BC8-D0701176C374](https://user-images.githubusercontent.com/62087923/76574846-b5605b00-647a-11ea-93b8-8ad5b9594132.png)

vs. now:

![9034432F-3297-4F7C-8588-5D0B9FD27327](https://user-images.githubusercontent.com/62087923/76574879-cad58500-647a-11ea-8a5c-b69d685fe31a.png)


I did not see you mention this in the changelog so it think it would be worth mentioning.

Thanks Anyway
